src/components/RecipeLinks.astro

```diff
@@ -0,0 +1,76 @@
+---
+import { getEntryBySlug } from 'astro:content';
+
+export interface Props {
+	slugs: string[];
+}
+
+function convertSlug(slug: string) {
+	let corrected = slug;
+	if (slug.startsWith('/')) {
+		corrected = corrected.slice(1);
+	}
+	if (slug.endsWith('/')) {
+		corrected = corrected.slice(0, -1);
+	}
+	return corrected;
+}
+
+const slugs = Astro.props.slugs.map(convertSlug);
+
+const entries = await Promise.all(
+	slugs.map(async (slug, i) => {
+		let entry = await getEntryBySlug('docs', slug);
+		if (!entry) throw Error(`Could not find entry for slug "${Astro.props.slugs[i]}"`);
+		return entry;
+	})
+);
+
+const isList = entries.length > 1;
+const noun = isList ? 'recipes' : 'recipe';
+---
+
+<div class="root">
+	<div class="flex">
+		<img src="/houston_chef.png" width="26" alt="Chef Houston" />
+		<span class="title">Related {noun}:</span>
+		{!isList && <a href={`/${entries[0]!.slug}/`}> {entries[0]!.data.title}</a>}
+	</div>
+	{
+		isList && (
+			<ul>
+				{entries.map((entry) => (
+					<li>
+						<a href={`/${entry!.slug}/`}>{entry!.data.title}</a>
+					</li>
+				))}
+			</ul>
+		)
+	}
+</div>
+
+<style>
+	.title {
+		font-weight: 600;
+		font-size: 1rem;
+		color: #c7c5d3;
+	}
+	.flex {
+		display: flex;
+		align-items: center;
```

Comment:
We should let items wrap here to avoid this effect on narrow viewports:
![image](https://user-images.githubusercontent.com/357379/234799117-7bf70d35-dc31-4c26-9bab-beb5b8af1c91.png)


```suggestion
		align-items: center;
		flex-wrap: wrap;
```